internal/go_repository.bzl

```diff
@@ -84,6 +84,7 @@ def _go_repository_impl(ctx):
     else:
         fail("one of urls, commit, tag, or importpath must be specified")
 
+    generate = ctx.attr.build_file_generation == "on"
```

Comment:
Thanks for looking at this, but this fix isn't sufficient. The whole block where `generate` is defined should move up.

Reply:
Thanks; I should have noticed. Done. PTAL.